spec/manageiq/api/action_result_spec.rb

```diff
@@ -0,0 +1,76 @@
+describe ManageIQ::API::Client::ActionResult do
+  let(:invalid_action_result_error) { "Not a valid Action Result specified" }
+  let(:success_action_result) { {"success" => true, "message" => "Vm powered on"} }
+  let(:failed_action_result) { {"success" => false, "message" => "Vm could not be powered on"} }
+
+  describe "#new" do
```

Comment:
Same goes for all of the other class methods below. `#` indicates an instance method.